app/experimenter/nimbus-ui/src/components/TableVisualizationRow/index.tsx

```diff
@@ -0,0 +1,206 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+
+// import ConfidenceInterval from "../ConfidenceInterval";
+import {
+  SIGNIFICANCE,
+  // METRIC,
+  BRANCH_COMPARISON,
+  TABLE_LABEL,
+  VARIANT_TYPE,
+  DISPLAY_TYPE,
+} from "../../lib/visualization/constants";
+import { SIGNIFICANCE_TIPS } from "../../lib/visualization/constants";
+import { BranchDescription } from "../../lib/visualization/types";
+import { ReactComponent as SignificanceNegative } from "./significance-negative.svg";
+import { ReactComponent as SignificancePositive } from "./significance-positive.svg";
+import { ReactComponent as SignificanceNeutral } from "./significance-neutral.svg";
+
+// This is a mapping for which view on the analysis
+// to display given the branch and table type.
+const dataTypeMapping = {
+  [TABLE_LABEL.RESULTS]: {
+    [VARIANT_TYPE.CONTROL]: BRANCH_COMPARISON.ABSOLUTE,
+    [VARIANT_TYPE.VARIANT]: BRANCH_COMPARISON.ABSOLUTE,
+  },
+  [TABLE_LABEL.HIGHLIGHTS]: {
+    [VARIANT_TYPE.CONTROL]: BRANCH_COMPARISON.ABSOLUTE,
+    [VARIANT_TYPE.VARIANT]: BRANCH_COMPARISON.UPLIFT,
+  },
+  [TABLE_LABEL.PRIMARY_METRICS]: {
+    [VARIANT_TYPE.CONTROL]: BRANCH_COMPARISON.ABSOLUTE,
+    [VARIANT_TYPE.VARIANT]: BRANCH_COMPARISON.ABSOLUTE,
+  },
+};
+
+const showSignificanceField = (
+  significance: string | undefined,
+  interval: string,
+  name: string,
+  tableLabel: string,
+) => {
+  let significanceIcon,
+    changeText = "";
+  // Attributes set to 'undefined' don't render in the DOM
+  const className = significance ? `${significance}-significance` : undefined;
+  switch (significance) {
+    case SIGNIFICANCE.POSITIVE:
+      significanceIcon = (
+        <SignificancePositive data-tip={SIGNIFICANCE_TIPS.POSITIVE} />
+      );
+      changeText = "increased";
+      break;
+    case SIGNIFICANCE.NEGATIVE:
+      significanceIcon = (
+        <SignificanceNegative data-tip={SIGNIFICANCE_TIPS.NEGATIVE} />
+      );
+      changeText = "decreased";
+      break;
+    case SIGNIFICANCE.NEUTRAL:
+      significanceIcon = (
+        <SignificanceNeutral data-tip={SIGNIFICANCE_TIPS.NEUTRAL} />
+      );
+      changeText = "is similar to control";
+      break;
+  }
+
+  let intervalText = `(${interval} change)`;
+  if (!significance) {
+    intervalText = `(${interval})`;
+  }
+
+  return (
+    <>
+      <p {...{ className }} data-testid={className}>
+        {significanceIcon}
+        &nbsp;
+        {tableLabel === TABLE_LABEL.HIGHLIGHTS
+          ? `${name} ${changeText} ${intervalText}`
+          : `${interval}`}
+      </p>
+      <ReactTooltip />
```

Comment:
@emtwo I replaced the tooltips for this table to better reflect Figma:

<img width="1039" alt="image" src="https://user-images.githubusercontent.com/13018240/99326842-42478980-283e-11eb-9d63-651b8684e840.png">